src/app/moviedb-collections/moviedb-popular-api.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, forkJoin } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { MovieCollectionDTO, MovieItem, SeriesCollectionDTO } from './moviedb-collection';
+import { Moviedb } from '../infrastucture/http/moviedb-http';
+import { formatResponse } from '../helpers/format-response';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MoviedbPopularApiService {
+  constructor(private readonly http: HttpClient) {}
+
+  private getPopularMovies(type: string): Observable<MovieCollectionDTO[]> {
+    return this.http.get<Moviedb<MovieCollectionDTO[]>>(`/movie/${type}`).pipe(map((response) => response.results));
+  }
+  private getPopularSeries(type: string): Observable<SeriesCollectionDTO[]> {
+    return this.http.get<Moviedb<SeriesCollectionDTO[]>>(`/tv/${type}`).pipe(map((response) => response.results));
+  }
+
+  getPopular(type: string): Observable<MovieItem[]> {
+    return forkJoin([this.getPopularMovies(type), this.getPopularSeries(type)]).pipe(
+      map((response) => formatResponse([...response[0], ...response[1]]))
```

Comment:
nit: You can use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here.

Something like that
```
map(([movies, series]) => formatResponse([...movies, ...series]))
```